packages/types/responses.ts

```diff
@@ -245,7 +245,7 @@ export const ZResponseMeta = z.object({
     .optional(),
   country: z.string().optional(),
   action: z.string().optional(),
-  ipAddress: z.string().optional(),
+  ip: z.string().optional(),
```

Comment:
🔧 **Fix**: We should add .ip() here:

```suggestion
  ip: z.string().ip().optional(),
```

Reply:
import { ZIP } from "@formbricks/types/common";
ZIP is added newly and used while parsing in extractIP function, anyhow added here